storage/innobase/xtrabackup/src/xtrabackup.cc

```diff
@@ -906,6 +909,13 @@ struct my_option xb_client_options[] =
    (uchar *) &opt_noversioncheck,
    0, GET_BOOL, NO_ARG, 0, 0, 0, 0, 0, 0},
 
+  {"no-tables-compatibility-check", OP_XTRA_NO_TABLES_COMPATIBILITY_CHECK,
```

Comment:
Lets rename this option to tables_compatibility_check with default ON, then since it is boolean valid syntax will be for example:
--skip-tables-compatibility-check
and
--tables-compatibility-check=NO